docs/getstarted/telemetry.md

```diff
@@ -66,6 +66,33 @@ If you'd like to view all the possible telemetry events that VS Code could send,
 
 For example, running `code --telemetry > telemetry.json && code telemetry.json` will create a `telemetry.json` file in your current working directory and then open it in VS Code. You cannot pipe the output like this, `code --telemetry | code -`, due to the length of the telemetry report.
 
+When events are created, metadata provides details about telemetry classification, purpose, and any special handling. The sections below describe these fields.
```

Comment:
This sentence sounds off.
Maybe "The section below details the event metadata used to classify our telemetry, define its purpose, and indicate any special handling"

Reply:
Agreed, it did come out odd